docs/conf.py

```diff
@@ -40,6 +40,40 @@
     "http://cs231n.stanford.edu/"
 ]
 
+html_output_directory = "_build/html"
+setting_all_article_info = True
+all_article_info_os = ["linux", "windows"]
+all_article_info_author = ""
+all_article_info_date = "May 1, 2023"
```

Comment:
I was under the impression you were getting the date with  : :date: "{sub-ref}`today`" and time with : :read-time: "{sub-ref}`wordcount-minutes` min read"

From https://sphinx-design.readthedocs.io/en/sbt-theme/additional.html

Reply:
MyST substitutions require myst-parser version 1.0.0, but our myst-nb dependency requires myst-parser to be version 0.18 (https://github.com/executablebooks/MyST-NB/blob/master/pyproject.toml)

My workaround was to insert the HTML into the output, which has the added benefit of being independent of the source language (i.e.: RST or Markdown) and avoids having to replace an article info header if it already exists in the source file